### DISCUSSION TITLE
hi  can you please help me in solving this error

### DISCUSSION THREAD

**Discussion by raju-b70** [learndevopsonline/learndevopsonline#1850] [Q&A]
![Screenshot (34)](https://github.com/learndevopsonline/learndevopsonline/assets/161826606/c895d0f8-7c04-4242-a2cb-cb5dc83396f2)

**Comment by chaithanya1812** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/efa346be-953b-4284-b796-94a52401659e)
## No server Ip is attached for this  Dns-Record..
